### ISSUE TITLE
Not able to use multiple files with the tool.

### ISSUE THREAD

**Issue by sangee2004** [gptscript-ai/gpt4-v-vision#1]
**Steps to reproduce the problem:**

1. Clone this repo locally.
2. Tried to execute the following gptscript with multiple files
```
tools: tool.gpt

Describe the picture in files examples/eiffel-tower.png, examples/wes.gpt with high fidelity using prompt as describe the picture in detail
``` 

Following errors are seen:

```
17:11:05 started  [main]
17:11:05 sent     [main]
17:11:05 started  [vision(2)] [input={"detail":"high","images":"file://~/Downloads/sunrise.jpeg","prompt":"describe the picture in detail"}]
17:11:05 sent     [vision(2)]
(node:99589) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
node:fs:581
  return binding.open(
                 ^

Error: ENOENT: no such file or directory, open '~/Downloads/sunrise.jpeg'
    at Object.openSync (node:fs:581:18)
    at Object.readFileSync (node:fs:460:35)
    at resolveImageURL (file:///Users/sangeethahariharan/vision/index.js:89:23)
    at Command.run (file:///Users/sangeethahariharan/vision/index.js:58:20)
    at Command.listener [as _actionHandler] (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:482:17)
    at /Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1283:27)
    at Command.parseAsync (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:935:16)
    at main (file:///Users/sangeethahariharan/vision/index.js:48:17) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '~/Downloads/sunrise.jpeg'
}

Node.js v21.6.2
17:11:05 failed to run tool [vision] cmd [/bin/bash /var/folders/xt/zg5q3qtj07q84xvf0_80pfdh0000gn/T/gptscript2775371251]: exit status 1
2024/02/22 17:11:05 ERROR: (node:99589) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
node:fs:581
  return binding.open(
                 ^

Error: ENOENT: no such file or directory, open '~/Downloads/sunrise.jpeg'
    at Object.openSync (node:fs:581:18)
    at Object.readFileSync (node:fs:460:35)
    at resolveImageURL (file:///Users/sangeethahariharan/vision/index.js:89:23)
    at Command.run (file:///Users/sangeethahariharan/vision/index.js:58:20)
    at Command.listener [as _actionHandler] (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:482:17)
    at /Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:1283:27)
    at Command.parseAsync (/Users/sangeethahariharan/vision/node_modules/commander/lib/command.js:935:16)
    at main (file:///Users/sangeethahariharan/vision/index.js:48:17) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '~/Downloads/sunrise.jpeg'
}

Node.js v21.6.2
: exit status 1
```

**Comment by njhale** [comment]
I think this may have been resolved by #20. @sangee2004 could you try again when you get the chance?

**Comment by sangee2004** [comment]
This issue is not seen anymore when testing with latest version of the tool ( as remote tool).
